apps/wildfires/src/app/pages/introduction/firesSurvey/config/forms/questions/incomeAssessment.tsx

```diff
@@ -33,7 +33,7 @@ export const qIncomeAssessment: TQuestion = {
   title:
     'Based on your household size, is your income LESS or GREATER than the number in this chart?',
   subtitle:
-    "Income includes things like wages, tips and government benefit payments. It's OK if you don't know your household's exact income.",
+    "Please see the chart below and tell us if, for the number of people in your household, your household’s total pre-tax income (before the fires) was less or greater than the income amount listed in that row of the chart?",
```

Comment:
no question mark, since this isn't a question
```suggestion
    "Please see the chart below and tell us if, for the number of people in your household, your household’s total pre-tax income (before the fires) was less or greater than the income amount listed in that row of the chart.",
```